### PR TITLE
Logistic request/bugfix - handling ErrorException - Trying to get property 'roles' of non-object

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
image

### CODE REVIEW

File: app/Http/Controllers/API/v1/LogisticRequestController.php [hunk 1]
```diff
@@ -87,7 +87,14 @@ public function show(Request $request, $id)
 
         $response = response()->format(Response::HTTP_OK, 'success', $data);
 
-        $isNotAdmin = !in_array(JWTAuth::user()->roles, User::ADMIN_ROLE);
+        $isNotAdmin = false;
+
+        try {
+            $isNotAdmin = !in_array(JWTAuth::user()->roles, User::ADMIN_ROLE);
+        } catch (\Exception $exception) {
+            return response()->format(Response::HTTP_UNAUTHORIZED, 'Anda tidak memiliki akses untuk membuka alamat URL tersebut.');
```
Comment (ayocodingit) [inline app/Http/Controllers/API/v1/LogisticRequestController.php:95]: this is for authorized check needs, why not use the laravel request form or gate authorize default? :smile:

**Review by ayocodingit — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/API/v1/LogisticRequestController.php (hunk 1 above)
Comment (sandisunandar99) [inline app/Http/Controllers/API/v1/LogisticRequestController.php:95]: why set _return response_ in catch exception?

Comment (firmanJS) [reply app/Http/Controllers/API/v1/LogisticRequestController.php:95]: interesting :1st_place_medal:

Comment (rindibudiaramdhan) [reply app/Http/Controllers/API/v1/LogisticRequestController.php:95]: interesting. This is fastest I can think. Okay I will try request Form Class @ayocodingit

Comment (rindibudiaramdhan) [reply app/Http/Controllers/API/v1/LogisticRequestController.php:95]: to denied access when user did not have roles @sandisunandar99

**Comment by rindibudiaramdhan** [comment]
sorry, I add fix to some Unit Test Class.

**Review by yohang88 — APPROVED** [review]

**Review by firmanJS — APPROVED** [review]

File: app/Http/Controllers/API/v1/LogisticRequestController.php (hunk 1 above)
Comment (rindibudiaramdhan) [reply app/Http/Controllers/API/v1/LogisticRequestController.php]: kang @ayocodingit , problem is this endpoint is not registered in the auth middleware. so it needs to be returned.
So in this class did not need change anymore.

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**